scout/models/case/case.py

```diff
@@ -54,7 +54,6 @@
     diagnosis_phenotypes=list,  # List of dictionaries with OMIM disease data
     display_name=str,  # required. This is the case name that will be shown in scout.
     dynamic_gene_list=list,  # List of genes
-    exe_ver=str,  # Path to the pipeline executable versions report file
```

Comment:
I think the key saved on the database for the case is pipeline_version, at least it's the one that is expected on the [formats constant](https://github.com/Clinical-Genomics/scout/blob/79e171597bfdadea788832c763b510817b4855f7/scout/constants/case_tags.py#L8)

Reply:
But hold on, this is still weird: if this is the db key, then we have a bunch of other places to change it as well, don't we? Like https://github.com/Clinical-Genomics/scout/blob/b0cfc8795392ed7e1b223eeaa5ad5590fc6e8892/scout/adapter/mongo/case.py#L1000?

Reply:
Ok it looks weird but if you try the command then it updates the path correctly, perhaps we can change the comments in there to be less confusing

Reply:
The cli command to update that field is not invoking that command it seems, but I fixed the adapter so that whenever you make another update in the case you keep also the pipeline_version